File: 03_private_encapculation.cpp
#include<bits/stdc++.h>

using namespace std;

class A{
    int value; // private here

    public:
        string name;
        A(string name){
            this->name = name;

        }
};

int main(){

    A obj("shakil");
    obj.value = 10;
    cout<<obj.value<<" "<<obj.name<<endl; 

    return 0;
}

File: 03_private_encapculation.py
class Person:
    def __init__(self, name, age) -> None:
        self.__name = name
        self.__age = age

    def __showDetail(self):
        print(f'My name is ',self.__name, 'and my age is ',self.__age)

p1 = Person('Shakil',22)
p1._Person__showDetail()